doc/application/coccinelle.rst

```diff
@@ -513,8 +515,8 @@ Coccinelle Mailing List
 ***********************
 
 Interested developers can subscribe coccinelle mailing list via:
-        https://systeme.lip6.fr/mailman/listinfo/cocci
+* https://systeme.lip6.fr/mailman/listinfo/cocci
```

Comment:
a blank line is needed before the first item in a bullet list

Also, on the line above, change to ``subscribe to the coccinelle mailing list``